gateway-ha/src/test/resources/rules/routing_rules_state.yml

```diff
@@ -0,0 +1,31 @@
+---
+name: "initialize state"
+description: "Add a set to the state map to track rules that have evaluated to true"
+priority: 0
+condition: "true"
+actions:
+  - |
```

Comment:
Is `- |` required, or is that some kind of coding style?

Reply:
The `-` indicates this is an array element, the `|` indicates the beginning of a block scalar: https://github.com/yaml/yaml-grammar/blob/75083c2aaabd0d7453d72de895666b21021826c1/yaml-spec-1.2.txt#L1689

In this instance I'm using it because it allows me to avoid escaping all of the quotes in lines like
```
state.get("triggeredRules").contains("airflow") && request.getHeader("X-Trino-Client-Tags") contains "label=special"
```